### PR TITLE
10.0 mig sale generator

### PR DESCRIPTION
**Pull request by EBII**
Allow to generate multiple sales at once

### CODE REVIEW

File: sale_generator/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Sale Generator
+==============
+
+This module allows to generate multiple sales order at once.
+
+.. figure:: path/to/local/image.png
```
Comment (legalsylvain) [inline sale_generator/README.rst]: not a real path. Could you add a real screenshot ?

File: sale_generator/models/sale_generator.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        res = super(SaleGenerator, self).write(vals)
+        if 'partner_ids' in vals and self.state == 'generating':
+            self._update_order()
+        return res
+
+    def add_generated_partner(self):
+        return {'type': 'ir.actions.act_window',
+                'res_model': 'res.partner',
+                'name': u"New Customer",
+                # ressource associé à l id xml de l'action
```
Comment (legalsylvain) [inline sale_generator/models/sale_generator.py]: french sentence.

File: sale_generator/views/generator_view.xml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                        icon="fa-strikethrough"
+                        string="Sales"/>
+                <group name="header" colspan="2">
+                    <field name="date_order"/>
+                    <field name="tmpl_sale_id"
+                           context="{'default_is_template': True, 'default_name': '', 'default_partner_id': %(sale_generator.res_partner_for_sale_template)d}"/>
+                    <field name="warehouse_id"/>
+                </group>
+                <separator string="Customer"/>
+                <span>In order to add customer you have two possibility:
```
Comment (legalsylvain) [inline sale_generator/views/generator_view.xml]: possibilities

**Review by legalsylvain — COMMENTED** [review]
please git mv the next time, it make the review easier.
thanks !

**Comment by EBII** [comment]
@legalsylvain changes done  °<:0)

**Comment by mourad-ehm** [comment]
code review not tested. :+1:

**Comment by EBII** [comment]
git mv #464

**Pull request closed without merging**